### PR TITLE
MWPW-147149: Prevent multiple download/upgrade CTAs

### PR DESCRIPTION
**Pull request by yesil**
in the same marquee or merch-card

Includes: https://git.corp.adobe.com/wcms/tacocat.js/pull/582

Resolves: [MWPW-147149](https://jira.corp.adobe.com/browse/MWPW-147149)

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/?martech=off
- After: https://mwpw-147149--milo--yesil.hlx.page/drafts/ilyas/checkout-links?martech=off

IMS test URL: https://ims--milo--adobecom.hlx.live/drafts/ilyas/checkout-links?martech=off

**Screenshots**

Before:
image

After:
image

### CODE REVIEW

File: libs/deps/commerce.js
Comment (npeltier) [inline libs/deps/commerce.js:1]: please not merge this with a non develop commit

**Review by npeltier — APPROVED** [review]
lgtm assuming you update commerce from develop branch

**Review by Axelcureno — APPROVED** [review]

**Comment by Roycethan** [comment]
@yesil  for Photoshop considering as Card - the expectation is both Buy and Try CTA's to replace with Download, Same as in Marquee where both CTA's will be replced with Upgrade/Download.
But in the sample test - the photoshop has 2 Downloads:
https://ims--milo--adobecom.hlx.live/drafts/ilyas/checkout-links?martech=off# (Login with noronha+photoshop@adobetest.com/Adobe123#)
image

Expectation:
image

**Comment by yesil** [comment]
@Roycethan this PR addresses only the marquee and merch-card blocks.

We don't have a strategy for regular inline buttons outside those blocks.

So the following is expected as it is outside any block.

image

**Review by Sartxi — APPROVED** [review]

**Pull request merged**
